stereum-control-center-cli.sh

```diff
@@ -10,6 +10,38 @@ dialog_title="Stereum Control Center"
 stereum_config_file_path=/etc/stereum/ethereum2.yaml
 
 function dialog_import_wallet() {
+  if [ "$setup" == "multiclient" ]; then
+  choice_launchpad_wallet_path=$(dialog --backtitle "$dialog_backtitle" \
+    --title "$dialog_title" \
+    --inputbox "Please enter the directory of the validator_keys\n(e. g. /tmp/validator_keys):" 9 60 "" \
+    3>&1 1>&2 2>&3)
+
+  choice_validator_number=$(dialog --backtitle "$dialog_backtitle" \
+    --title "$dialog_title" \
+    --inputbox "Please enter the number of the validator_keys:" 9 60 "" \
+    3>&1 1>&2 2>&3)
+
+  choice_account_password=$(dialog --backtitle "$dialog_backtitle" \
```

Comment:
See comment in review of https://github.com/stereum-dev/ethereum2-ansible/pull/37 about using the same auto generated passphrase for both: wallet and account

Reply:
var `validator_password` also `password dialog` is removed. Create-account & wallet going to use `auto-generated` passphrase